### PR TITLE
Build/netflix login page

### PR DESCRIPTION
**Pull request by aguilarjayson19300**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import './App.css';
+import './style/style.css';
+import './style/font.css';
+import './style/icon.css';
+import { InputField } from './components';
+import { languages } from './data';
+// import {languages} from './data'
```
Comment (edmondswrtech) [inline src/App.tsx]: remove comment

File: src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        <header className="header">
+          <img src="/logo.png" alt="logo" />
+        </header>
+        <div className="registerBlock">
+          <div className="paddingBlock">
+            <div className="title">
+              <h1>Sign In</h1>
+            </div>
+
+            <div className="inputs">
```
Comment (edmondswrtech) [inline src/App.tsx]: be specific in naming style

File: src/App.tsx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+                This page is protected by Google reCAPTCHA to ensure you're not a bot. <span className="learnMore">Learn more.</span>
+              </span>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <footer className="footer">
+        <div className="footerContent">
+          <div className="block1">
```
Comment (edmondswrtech) [inline src/App.tsx]: be specific in naming style

File: src/App.tsx [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+            <span>Question?</span>
+            <span>Contact us.</span>
+          </div>
+
+          <div className="block2">
```
Comment (edmondswrtech) [inline src/App.tsx]: also here

File: src/App.tsx [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+            <a href="#">FAQ</a>
+            <a href="#">Help Center</a>
+            <a href="#">Term of use</a>
+            <a href="#">Privacy</a>
+          </div>
+
+          <div className="block3">
```
Comment (edmondswrtech) [inline src/App.tsx]: also here naming

File: src/Home.jsx [hunk 6]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export default function Home() {
```
Comment (edmondswrtech) [inline src/Home.jsx]: use arrow function

File: src/pages/login.tsx [hunk 7]
```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import './App.css';
+import './style/style.css';
+import './style/font.css';
+import './style/icon.css';
+import { InputField } from '../components';
+import { languages } from '../data';
+
+
+const login = () => {
```
Comment (edmondswrtech) [inline src/pages/login.tsx]: make the function name capital

File: src/pages/login.tsx [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+  return (
+    <body style={{
+        backgroundImage: `url("bgdar.png")`
+      }} >
+        <div className="container">
+          <header className="header">
+            <img src="/logo.png" alt="logo" />
+          </header>
+          <div className="registerBlock">
+            <div className="paddingBlock">
```
Comment (edmondswrtech) [inline src/pages/login.tsx]: dont make the style name of your variable

File: src/pages/login.tsx [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+              <div className="title">
+                <h1>Sign In</h1>
+              </div>
+  
+              <div className="inputs">
```
Comment (edmondswrtech) [inline src/pages/login.tsx]: change style name

File: src/pages/login.tsx [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+                <InputField
+                  type="text"
+                  placeHolder="Email or Phone number"
+                />
+                <InputField
+                  type="password"
+                  placeHolder="Password"
+                />
+              </div>
+
```
Comment (edmondswrtech) [inline src/pages/login.tsx]: remove space

File: src/pages/login.tsx [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+              </div>
+  
+              <div className="button">
+                <button>Sign In</button>
+                <div className="checkboxBlock">
+                  <div className="checkbox">
+                    <input type="checkbox" />
+                    <span>Remember me</span>
+                  </div>
+
```
Comment (edmondswrtech) [inline src/pages/login.tsx]: also here

File: src/pages/login.tsx [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+                    <span>Remember me</span>
+                  </div>
+  
+                  <div>
+                    <span className="isHelp">Need help?</span>
+                  </div>
+  
+                </div>
+              </div>
+
```
Comment (edmondswrtech) [inline src/pages/login.tsx]: remove space

File: src/pages/login.tsx [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+                  This page is protected by Google reCAPTCHA to ensure you're not a bot. <span className="learnMore">Learn more.</span>
+                </span>
+              </div>
+            </div>
+          </div>
+        </div>
+  
+        <footer className="footer">
+          <div className="footerContent">
+            <div className="block1">
```
Comment (edmondswrtech) [inline src/pages/login.tsx]: change name

File: src/pages/login.tsx [hunk 14]
```diff
@@ -0,0 +1,102 @@
...
+              <span>Question?</span>
+              <span>Contact us.</span>
+            </div>
+  
+            <div className="block2">
```
Comment (edmondswrtech) [inline src/pages/login.tsx]: also here

File: src/pages/login.tsx [hunk 15]
```diff
@@ -0,0 +1,102 @@
...
+              <a href="#">FAQ</a>
+              <a href="#">Help Center</a>
+              <a href="#">Term of use</a>
+              <a href="#">Privacy</a>
+            </div>
+  
+            <div className="block3">
```
Comment (edmondswrtech) [inline src/pages/login.tsx]: also here

File: src/data.tsx [hunk 16]
```diff
@@ -0,0 +1,5 @@
+export const languages = [
```
Comment (edmondswrtech) [inline src/data.tsx]: be specific in naming

File: src/components/InputField.tsx [hunk 17]
```diff
@@ -0,0 +1,23 @@
...
+export const InputField = ({ onChangeText = () => null, inputValue, placeHolder, type, id }: InputFieldProps) => {
+    return (
+        <input
+            type={type}
+            placeholder={placeHolder}
+            onChange={(e) => {
+                onChangeText(e.target.value);
+            }}
+            value={inputValue}
+        ></input>
```
Comment (edmondswrtech) [inline src/components/InputField.tsx]: already close the tag

File: src/components/InputField.tsx [hunk 18]
```diff
@@ -0,0 +1,23 @@
+
+interface InputFieldProps {
+    onChangeText?: (text: string) => void;
+    inputValue?: string;
+    placeHolder?: string;
+    type?: string;
+    id?: String;
```
Comment (edmondswrtech) [inline src/components/InputField.tsx]: Why all of the property is optional?

**Review by edmondswrtech — CHANGES_REQUESTED** [review]
use proper naming convention and remove unused codes

File: src/data.tsx (hunk 16 above)
Comment (aguilarjayson19300) [reply src/data.tsx]: As per Mika Advise nothing to resolve to this issue

**Review by edmondswrtech — APPROVED** [review]

File: src/components/InputField.tsx [hunk 19]
```diff
@@ -0,0 +1,23 @@
+
+interface InputFieldProps {
```
Comment (mika-ella) [inline src/components/InputField.tsx:1]: why is everything optional? should have a required property

File: src/components/Navbar.tsx [hunk 20]
```diff
@@ -0,0 +1,67 @@
...
+    const [showOption, setShowOption] = useState(false);
+
+    const handleMouseEnter = () => {
+      setShowOption(true);
+    };
+  
+    const handleMouseLeave = () => {
+      setShowOption(false);
+    };
+    //========================================================================
```
Comment (mika-ella) [inline src/components/Navbar.tsx]: remove

File: src/pages/home.tsx [hunk 21]
```diff
@@ -0,0 +1,42 @@
...
+import '../style/font.css';
+import '../style/icon.css';
+
+export const Home = () => {
+  return (
+
+    <div className='home'>
+      <Navbar />
+      <Featured />
+      <br></br><br></br><br></br><br></br><br></br><br></br><br></br><br></br>
```
Comment (mika-ella) [inline src/pages/home.tsx]: do not use break element to add space, use padding or margin

File: src/data.tsx (hunk 16 above)
Comment (mika-ella) [inline src/data.tsx]: create interface for languages

**Review by mika-ella — CHANGES_REQUESTED** [review]
resolve all comments

**Review by jarapamikaella — APPROVED** [review]

**Review by mika-ella — APPROVED** [review]

**Pull request merged**